### DISCUSSION TITLE
RD Windows User - Environment Information

### DISCUSSION THREAD

**Discussion by evertonlperes** [rancher-sandbox/rancher-desktop#940] [Q&A]
We're looking to collect more information regarding Windows environments in order to help us during the Rancher Desktop debug.

We'd like to hear from you on the following items that you might being using on your Windows env:

* Windows Version
* Windows [build](https://docs.microsoft.com/en-us/windows/release-health/release-information#windows-10-current-versions-by-servicing-option) version
* VPN type
* Proxy
* Firewall [rules](https://docs.microsoft.com/en-us/powershell/module/netsecurity/get-netfirewallrule?view=windowsserver2019-ps#examples)
* Special Activity directory features
* 3rd party Anti-Virus / Security Software
* Any other relevant information about your Windows Env.

Here's a example that you should use to fill up and paste in this thread:

* Windows Version: `Home`
* Windows build version: `1909` 
* VPN type: `Pulse Secure`
* Proxy: `N/A`
* Firewall rules: `N/A`
* Special Activity directory features: `Universal scope - no custom configuration`
* 3rd party Anti-Virus/Security Software: `Kaspersky 21.2x`

Note 1: please, use the example above (copy -> edit -> paste) as a comment on this thread. 
Note 2: We also understand some of the information could be managed by your IT group and it may not be possible to share due company rules.
 

We really appreciate your help.

**Comment by malcolmlewis** [comment]
Hi
Information as requested I'm using v0.6.1 of Rancher Desktop just in case you need that info as well.....

Windows Version: Windows 10 Pro
Windows build version: 19043.1288
VPN type: N/A
Proxy: N/A
Firewall rules: N/A
Special Activity directory features: Universal scope - no custom configuration
3rd party Anti-Virus/Security Software: N/A Windows Defender

**Comment by jpennell** [comment]
Windows Version: 10 Enterprise
Windows build version: 10.0.19043 Build 19043
VPN type: Cisco AnyConnect 4.10
Proxy: N/A
Firewall rules: N/A
Special Activity directory features: Universal scope - no custom configuration
3rd party Anti-Virus/Security Software: SentinelOne
